jerry-core/ecma/operations/ecma-function-object.c

```diff
@@ -266,12 +267,12 @@ ecma_object_check_constructor (ecma_object_t *obj_p) /**< ecma object */
  * @return ECMA_IS_VALID_CONSTRUCTOR - if the input value is a constructor.
  *         any other value - if the input value is not a valid constructor, the pointer contains the error message.
  */
```

Comment:
The "any other" value" comment is now invalid, no pointer is returned here. Please update.